lib/fix.js

```diff
@@ -1,11 +1,9 @@
-const lint = require('./lint');
-
 // (editor: Object) => function
-function fix(editor) {
+function fix(editor, lint) {
```

Comment:
Passing in a lint function, we can replace it in unit tests with the non `Task` version. While using the `Task` based version in the atom specs. I'll be doing this in a followup PR